packages/bot-skeleton/src/services/api/appId.js

```diff
@@ -1,35 +1,177 @@
-import { LiveApi } from 'binary-live-api';
-import { getLanguage } from '@deriv/translations';
-import AppIds from './appIdResolver';
+const DerivAPIBasic = require('@deriv/deriv-api/dist/DerivAPIBasic');
+const getAppId = require('@deriv/shared').getAppId;
+const getSocketURL = require('@deriv/shared').getSocketURL;
+const cloneObject = require('@deriv/shared').cloneObject;
+const getPropertyValue = require('@deriv/shared').getPropertyValue;
+const State = require('@deriv/shared').State;
+const { getLanguage } = require('@deriv/translations');
+const website_name = require('@deriv/shared').website_name;
 
-const getCustomEndpoint = () => ({
-    url: localStorage.getItem('config.server_url'),
-    appId: localStorage.getItem('config.app_id'),
-});
+/*
+ * An abstraction layer over native javascript WebSocket,
+ * which provides additional functionality like
+ * reopen the closed connection and process the buffered requests
+ */
+const BinarySocketBase = (() => {
+    let deriv_api, binary_socket;
 
-const isRealAccount = () => {
-    const active_loginid = localStorage.getItem('active_loginid') || '';
-    const is_real = typeof active_loginid === 'string' && !active_loginid.startsWith('VRT');
+    let config = {};
+    let wrong_app_id = 0;
+    let is_available = true;
+    let is_disconnect_called = false;
+    let is_connected_before = false;
 
-    return is_real;
-};
+    const getSocketUrl = () =>
+        `wss://${getSocketURL()}/websockets/v3?app_id=${getAppId()}&l=${getLanguage()}&brand=${website_name.toLowerCase()}`;
 
-const getDomainAppId = () => AppIds[document.location.hostname.replace(/^www./, '')];
+    const isReady = () => hasReadyState(1);
 
-const getDefaultEndpoint = () => ({
-    url: isRealAccount() ? 'green.binaryws.com' : 'blue.binaryws.com',
-    appId: localStorage.getItem('config.default_app_id') || getDomainAppId() || 19111,
-});
+    const isClose = () => !binary_socket || hasReadyState(2, 3);
```

Comment:
```suggestion
    const isClosed = () => !binary_socket || hasReadyState(2, 3);
```

Reply:
the appId content completely removed and now we use Basic type. of binary APIs